transcribed/page0120.txt

```diff
@@ -17,10 +17,10 @@
 017
 018     001 018 USTRA:  MOVEI U,0       ;-1(P) MUST BE U
 019
-020     117 014 USTR3:  CAML U,USRHI
+020     117 014 USTR3:  CAML U,USRHI    ;ASSIGN MORE CORE TO USER STORAGE
```

Comment:
This comment should be on the next line.

Reply:
Yes.